wfe/wfe.go

```diff
@@ -1788,6 +1792,8 @@ func (wfe *WebFrontEndImpl) Order(
 		}
 	}
 
+	response.Header().Add("Retry-After", strconv.Itoa(wfe.retryAfterOrder))
```

Comment:
Since `Retry-After` [may be represented by a seconds duration or by an absolute date](https://datatracker.ietf.org/doc/html/rfc7231#section-7.1.3), perhaps it would be good to randomize the representation here.